doc/InteroperabilityGuide.md

```diff
@@ -0,0 +1,64 @@
+Use *cupla* together with other alpaka based software
```

Comment:
Forgot to capitalize first letters?

Reply:
I'm not sure, if we official write `cupla` lower case like `alpaka`. But in this documentation, it is also written lower case: https://github.com/alpaka-group/cupla/blob/master/doc/TuningGuide.md

Reply:
Cupla yes, but `based` and `software`

Reply:
Ah, I understand.